### PR TITLE
More Typos

### PR DESCRIPTION
**Pull request by Cyrano7**
'It's' should only be used as 'it is' or 'it has' not for possessives.

### CODE REVIEW

File: The_Seven_Kingdoms/localisation/001_religion_descs.csv [hunk 1]
```diff
@@ -3,8 +3,8 @@ the_seven_DESC;The Faith of the Seven, or the New Gods, is the primary religion
...
+the_smith_desc;The Smith is the further worship of one of the tenets of the Faith of the Seven, as its own monotheistic religion. Followers of the Smith believe he will bless them with aid on large projects, and that they will benefit from his power in their daily lives.\n\n This imbalance has caused them to be seen as heretical in the eyes of the Most Devout.;;;;;;;;;;;x
+lady_of_the_waves_DESC;The Lady of the Waves is the polytheistic religion of the Sistermen. It was one of four faiths followed by the First Men, but unlike the other three, it did not survive the Andal invasion of Westeros. This faith believed in two deities, The Lady of the Waves, and the Lord of the Skies. Little is known of the religion, or its practices and myths, but it was said that storms, which were believed to be sacred, were the result of the Lady of the Waves mating with the Lord of the Skies.;;;;;;;;;;;;;x
```
Comment (Cyrano7) [inline The_Seven_Kingdoms/localisation/001_religion_descs.csv:7]: Fixed spelling of deities and its

File: The_Seven_Kingdoms/localisation/005_unique_buildings.csv [hunk 2]
```diff
@@ -77,7 +77,7 @@ tb_wildling_barracks3_desc;A large building to hold wildling troops.;;;;;;;;;;;;
...
-ca_the_arbor_desc;The Arbor is a golden island off the southwestern most part of Westeros, ruled by House Redwyne. It is considered part of the Reach, and is seperated from the mainland by the Redwyne Straits. It is protected by the Redwyne Fleet, the largest fleet of the Seven Kingdoms. The Arbor is known for making the best wine in Westeros, especially a terrific red one and a gold wine. Wine from the Arbor is usually sweeter than wine from Dorne. Gylbert of the Vines, the legendary founder of House Redwyne, is said to have taught the Arbor's people to make wine.;;;;;;;;;;;;;x
+ca_the_arbor_desc;The Arbor is a golden island off the southwestern most part of Westeros, ruled by House Redwyne. It is considered part of the Reach, and is separated from the mainland by the Redwyne Straits. It is protected by the Redwyne Fleet, the largest fleet of the Seven Kingdoms. The Arbor is known for making the best wine in Westeros, especially a terrific red one and a gold wine. Wine from the Arbor is usually sweeter than wine from Dorne. Gylbert of the Vines, the legendary founder of House Redwyne, is said to have taught the Arbor's people to make wine.;;;;;;;;;;;;;x
```
Comment (Cyrano7) [inline The_Seven_Kingdoms/localisation/005_unique_buildings.csv:80]: Fixed spelling of separated

**Comment by tannerflick4** [comment]
I'll let you work on some more, if you would like, and then I can merge.

**Comment by Cyrano7** [comment]
Should be good to go now.

**Comment by Cyrano7** [comment]
@tannerflick4 This look good now? Files were actually pretty clean compared to some other projects I've seen.

**Pull request merged**